client/lib/plugins/utils.js

```diff
@@ -50,6 +50,7 @@ function filterNoticesBy( site, pluginSlug, log ) {
 	return false;
 }
 
+// @todo(saramarcondes) Is this safe to rename? Would plugins potentially depend on this function's current name?
```

Comment:
@kwight I noticed you made the most recent "substantial" (non-formatting/chore) change to this file. Do you happen to know the answer to my question in this comment? Is it safe to rename this `whiteListPluginData` function?

Reply:
From what I can tell, the only two uses are here in this file: the definition, and a single call. The tests will have to be updated, but other than that, I think it's good to go!

Reply:
Right on, thanks so much!